utils.go

```diff
@@ -52,7 +52,7 @@ func GetBootLoader() (boot string, err error) {
 }
 
 func GetSystemLogManager() (syslog string, err error) {
-	out, err := exec.Command("bash", "-c", "sudo lsof | grep /var/log/syslog | cut -f1 -d' '").Output()
+	out, err := exec.Command("bash", "-c", "sudo lsof +D /var/log/syslog | cut -f1 -d' '").Output()
```

Comment:
Isn't /var/log/syslog a file and not a directory? I think that -D wouldn't run if it is not a directory

Reply:
You are correct, I changed it.